internal/render/common/apparmorprofile.go

```diff
@@ -0,0 +1,63 @@
+package common
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"nebius.ai/slurm-operator/internal/naming"
+	"nebius.ai/slurm-operator/internal/values"
+
+	apparmor "sigs.k8s.io/security-profiles-operator/api/apparmorprofile/v1alpha1"
+)
+
+func RenderAppArmorProfile(cluster *values.SlurmCluster) *apparmor.AppArmorProfile {
+	if !cluster.NodeLogin.UseDefaultAppArmorProfile || !cluster.NodeWorker.UseDefaultAppArmorProfile {
+		return nil
+	}
+	return &apparmor.AppArmorProfile{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      naming.BuildAppArmorProfileName(cluster.Name, cluster.Namespace),
+			Namespace: cluster.Namespace,
+		},
+		Spec: apparmor.AppArmorProfileSpec{
+			Policy: generateAppArmorPolicy(cluster.Name, cluster.Namespace),
+		},
+	}
+}
+
+func generateAppArmorPolicy(clusterName, namespace string) string {
+	return fmt.Sprintf(`#include <tunables/global>
+
+profile %s flags=(attach_disconnected,mediate_deleted) {
+  include <abstractions/base>
```

Comment:
Let's also add relevant paths from NVIDIA deb packages mentioned [here](https://ubuntu.com/server/docs/nvidia-drivers-installation#p-97843-installing-the-user-space-drivers-and-the-driver-libraries).

I've taken a look already, and viewed these files this way:
```
apt download nvidia-driver-550-server 

dpkg-deb -I nvidia-driver-550-server_550.127.08-0ubuntu0.22.04.1_amd64.deb
# Deps:
# libnvidia-gl-550-server
# nvidia-dkms-550-server
# nvidia-dkms-550-server
# nvidia-kernel-common-550-server
# nvidia-kernel-common-550-server
# nvidia-kernel-source-550-server
# libnvidia-compute-550-server
# libnvidia-extra-550-server
# nvidia-compute-utils-550-server
# libnvidia-decode-550-server
# libnvidia-encode-550-server
# nvidia-utils-550-server
# xserver-xorg-video-nvidia-550-server
# libnvidia-cfg1-550-server
# libnvidia-fbc1-550-server

apt download \
        libnvidia-gl-550-server \
        nvidia-dkms-550-server \
        nvidia-dkms-550-server \
        nvidia-kernel-common-550-server \
        nvidia-kernel-common-550-server \
        nvidia-kernel-source-550-server \
        libnvidia-compute-550-server \
        libnvidia-extra-550-server \
        nvidia-compute-utils-550-server \
        libnvidia-decode-550-server \
        libnvidia-encode-550-server \
        nvidia-utils-550-server \
        xserver-xorg-video-nvidia-550-server \
        libnvidia-cfg1-550-server \
        libnvidia-fbc1-550-server

find . -name "*.deb" -exec dpkg -c {} \; | cut -c 49- | grep -v '/$' | sort -u
```